components/thumbnail/index.css

```diff
@@ -123,6 +123,10 @@ governing permissions and limitations under the License.
 			overflow: hidden;
 			border-radius: var(--mod-thumbnail-border-radius, var(--spectrum-thumbnail-border-radius));
 		}
+
+		& .spectrum-OpacityCheckerboard {
+			--spectrum-opacity-checkerboard-size: var(--spectrum-thumbnail-opacity-checkerboard-square- size);
+		}
```

Comment:
Looks like this is under the `is-focused` selector. Do we need it at all here, now that we've scoped the thumbnail token within the opacity checkerboard css? If I delete it entirely, I think everything is still rendering properly with the right styles...but double check me on that!

Reply:
Oops! Good call though — just dropped it entirely and everything looks good. ✨